lib/ts/recipe/session/faunadb/recipeImplementation.ts

```diff
@@ -53,6 +53,13 @@ export default class RecipeImplementation implements RecipeInterface {
         return faunaResponse.secret;
     };
 
+    getGlobalClaimValidators = function (
```

Comment:
This file has been removed from the base latest X.Y branch. Please merge into this